src/ex6/server/client/src/components/AddTask/item_client.js

```diff
@@ -0,0 +1,32 @@
+export class ItemClient {
+  getItems = async () => {
+    const response = await fetch("http://localhost:3042/items")
```

Comment:
use relative URLs when possible - '/items' should resolve just the same
this will make your code less buggy on other computers & production